imagetool/src/main/java/com/oracle/weblogic/imagetool/cli/menu/ImageOperation.java

```diff
@@ -173,14 +174,28 @@ private String handlePasswordOptions() throws IOException {
         return cmdBuilder;
     }
 
+    public String getTempDirectory() throws IOException {
+        Path tmpDir = Files.createTempDirectory(Paths.get(Utils.getBuildWorkingDir()), "wlsimgbuilder_temp",
```

Comment:
Do we still need the posix permission? - Since we no longer mounting volumes for execution of script.